builds/fabric8-che/plugins/che-plugin-bayesian-lang-server/src/main/java/org/eclipse/che/plugin/languageserver/bayesian/server/launcher/BayesianLanguageServerLauncher.java

```diff
@@ -72,4 +76,19 @@ protected Process startLanguageServerProcess(String projectPath) throws Language
         }
     }
 
+    public LanguageServerDescription getDescription() {
+        return DESCRIPTION;
+    }
+
+    private static LanguageServerDescription createServerDescription() {
+        return new LanguageServerDescription("org.eclipse.che.plugin.bayesian.languageserver", //
+                                             null, //
+                                             Arrays.asList( //
+                                             new DocumentFilter(BayesianLanguageServerModule.LANGUAGE_ID, //
```

Comment:
I don't think "bayesian" should be a language. The bayesian language server adds info to other "languages". For example, a java linter would add info to the "java" language, not define it's own "java_linter" language.